### ISSUE TITLE
stable hlo select

### ISSUE THREAD

**Issue by AleksKnezevic** [tenstorrent/tt-mlir#1009]
Add unit tests for stable hlo slice to tt-torch and tt-xla.

**Comment by LPanosTT** [comment]
Yeah I closed it here https://github.com/tenstorrent/tt-mlir/issues/1003

**Comment by AleksKnezevic** [comment]
Not quite so easy, I messed up the name :)

**Comment by AleksKnezevic** [comment]
duplicate of https://github.com/tenstorrent/tt-mlir/issues/640
